test/libsolidity/smtCheckerTests/array_members/push_2d_arg_1_unsafe.sol

```diff
@@ -9,6 +9,8 @@ contract C {
 		assert(a[0][a[0].length - 1] == y);
 	}
 }
+// ====
+// SMTIgnoreCex: yes
 // ----
-// Warning 3944: (162-177): CHC: Underflow (resulting value less than 0) happens here.
-// Warning 6328: (150-184): CHC: Assertion violation happens here.
+// Warning 3944: (162-177): CHC: Underflow (resulting value less than 0) happens here.\nCounterexample:\na = [[]]\nx = []\ny = 0\n\n\nTransaction trace:\nconstructor()\nState: a = []\nf([], 0)
+// Warning 6328: (150-184): CHC: Assertion violation happens here.\nCounterexample:\na = [[]]\nx = []\ny = 0\n\n\nTransaction trace:\nconstructor()\nState: a = []\nf([], 0)
```

Comment:
Ah, maybe we can't assume no counterexamples in the expectations... but wouldn't it be better if there weren't any? Otherwise it looks like they're tested, but they aren't... is it hard to just filter the counterexample when ``isoltest`` updates expectations as well?

Reply:
Well it's quite easy to also see that they are ignored, if the option `SMTIgnore: yes` is there. As said in the other comment, I would also rather have them in the file and just ignore in the comparison.